proto/kuksa/val/v2/types.proto

```diff
@@ -22,7 +22,12 @@ message Datapoint {
   google.protobuf.Timestamp timestamp = 1;
 
   oneof value_state {
+    // When reading - returned if the signal exists but has no value,
```

Comment:
Thanks for adding some needed documentation, just wondering if here is the right place.
I usually made API changes to https://github.com/SoftwareDefinedVehicle/kuksa-databroker/tree/feature/databroker-api-v2 and then rebase main-branch onto that one and finally dev-branch onto main-branch.
I am confused now with the development process.